app/src/main/java/com/duckduckgo/app/pixels/AppPixelName.kt

```diff
@@ -40,6 +40,15 @@ enum class AppPixelName(override val pixelName: String) : Pixel.PixelName {
     ONBOARDING_DAX_CTA_OK_BUTTON("m_odc_ok"),
     ONBOARDING_PRIVACY_SHIELD_BUTTON("m_onboarding_privacy_shield_button"),
 
+    ONBOARDING_OPTION_PRIVATE_SEARCH_SELECTED("m_onboarding_option_private_search_selected"),
```

Comment:
did we open a privacy triage for these ones?

Reply:
yes :)